File: 2942-find-words-containing-character/2942-find-words-containing-character.php
class Solution {

    /**
     * @param String[] $words
     * @param String $x
     * @return Integer[]
     */
    function findWordsContaining($words, $x) {
        $indexes = [];

        for($i = 0; $i <= count($words); $i++)
        {
            if(str_contains($words[$i], $x))
                $indexes[] = $i;
        }
        
        return $indexes;
    }
}

File: 2942-find-words-containing-character/2942-find-words-containing-character.cs
public class Solution {
    public IList<int> FindWordsContaining(string[] words, char x) {
        var indexes = new List<int>();

        for(int i = 0; i < words.Length; i++)
        {
            if(words[i].Contains(x))
            {
                indexes.Add(i);
            }
        }

        return indexes;
    }
}